File: mobile/lib/domain/models/event.dart
class Event {
  final int id;
  final String name;
  final String shortDescription;
  final String description;
  final String city;
  final String address;
  final DateTime startDate;
  final DateTime endDate;
  final String photoUrl;
  final String category;

  Event({
    required this.id,
    required this.name,
    required this.shortDescription,
    required this.description,
    required this.city,
    required this.address,
    required this.startDate,
    required this.endDate,
    required this.photoUrl,
    required this.category,
  });
}

File: Domain/Entities/Event.cs
﻿namespace Domain.Entities;

public class Event
{
    public int Id { get; set; }
    
    public string Name { get; set; }

    public string? ShortDescription { get; set; }

    public string Description { get; set; }
    
    public Guid? CoverId { get; set; }
    public UserFile? Cover { get; set; }

    public string City { get; set; }

    public string Address { get; set; }

    public DateTimeOffset From { get; set; }
    
    public DateTimeOffset To { get; set; }
    
    public string OwnerId { get; set; }
    public User Owner { get; set; }
    
    public int? CategoryId { get; set; }
    public Category? Category { get; set; }
    
    public string CryptoKey { get; set; }
    
    public List<Ticket> Tickets { get; set; }
    
    public List<Question> Questions { get; set; }
    
    public List<User> Inspectors { get; set; }
}